modules/social_features/social_user/social_user.post_update.php

```diff
@@ -56,6 +56,9 @@ function social_user_post_update_10101_add_verified_role_to_existing_users(array
       // Add role "verified".
       $account->addRole('verified');
       $account->save();
+      if (\Drupal::moduleHandler()->moduleExists('ldap_user')) {
```

Comment:
Hi @Corn696 

The `ldap` module is not used in the distro.
Could you explain why you suggest these changes?

Thanks!

Reply:
Hi, we use the ldap module. Sure i could add this code temporarily and run the updates but i would guess it is a common use case to use ldap and the update should work for others as well or not?

Reply:
I think, in this case, you can create and attach a patch to the issue on drupal.org which others can use.

Reply:
Not optimal but i will do so. Thank you anyway.

Reply:
No problem just wanted to help. Seems that there was no progress on the ldap_user module regarding the issue. I made a patch as a workaround.